tests/integration/base.yaml

```diff
@@ -19,6 +19,7 @@ spec:
       env:
         UNSAFE_ENABLE_ADMIN_ENDPOINTS: true
         PILOT_REMOTE_CLUSTER_TIMEOUT: 15s
+        PILOT_ENABLE_ALPHA_GATEWAY_API: "true"
```

Comment:
Same comment - can we run the tests with the intended beta APIs ? I don't know what we cover by testing the alpha - the risks are in clusters having the beta CRDs AFAIK.

Reply:
This is so we can test the alpha resources. Which are ONLY alpha